test/features/comments.feature

```diff
@@ -0,0 +1,76 @@
+@smoke
+Feature: Comments
+
+    Scenario: DQL-74 BDD-1a Dataset comment submission access (non-logged in user)
```

Comment:
In BDD, we phrase everything in plain form, not formal.
`Logged-in user does not have an access to submit a comment`